### PR TITLE
Feature/docker with fresh sdk

### PR DESCRIPTION
**Pull request by vitalrev**

### CODE REVIEW

File: nodejs/indy/src/did/index.js [hunk 1]
```diff
@@ -78,17 +78,20 @@ exports.getTheirEndpointDid = async function (theirDid) {
     return metadata.theirEndpointDid;
 };
 
+//TODO: replace whatever with real user password
 async function setupSteward() {
     let stewardWalletName = `stewardWalletFor:${config.walletName}`;
     try {
-        await sdk.createWallet(config.poolName, stewardWalletName);
+        await sdk.createWallet({"id": stewardWalletName}, {"key": "whatever"});
```
Comment (saholman) [inline nodejs/indy/src/did/index.js]: Here we will also have to wait for indy-sdk 1.6 support in the node.js wrapper.

**Comment by saholman** [comment]
@vitalrev We can merge both of the PRs if you can change them to be compatible with the 1.5 syntax for creating wallets.  It may take some time for the node.js wrapper to be updated.

File: nodejs/indy/src/did/index.js (hunk 1 above)
Comment (vitalrev) [reply nodejs/indy/src/did/index.js]: yes

Comment (vitalrev) [reply nodejs/indy/src/did/index.js]: other question... if you look in the dockerfile... I clone indy-sdk (master), build libindy and build (npm install) indy node's wrapper. After that i install the builded indy-sdk wrapper into indy-agent. The idea is to use latest version of indy-sdk wrapper and libindy... Good for development, but bad from version management point of view

Comment (vitalrev) [reply nodejs/indy/src/did/index.js]: What we can do is, checkout a stable version of indy-sdk (eg 1.5) and build linindy and wrapper for this version... this is a good solution for a stable version (release) of indy-agent

**Comment by mhailstone** [comment]
Accepted ArthurManz update with Docker and 1.6.1.

**Pull request closed without merging**